src/components/tx-flow/flows/AddOwner/ChooseOwner.tsx

```diff
@@ -66,28 +91,60 @@ export const ChooseOwner = ({
             </Box>
           )}
 
-          <Box display="flex" flexDirection="column" gap={2} paddingTop={2}>
-            <Typography>New owner</Typography>
-            <FormControl>
-              <NameInput
-                label="Owner name"
-                name="name"
-                placeholder={fallbackName || 'New owner'}
-                InputLabelProps={{ shrink: true }}
-                InputProps={{
-                  endAdornment: resolving && (
-                    <InputAdornment position="end">
-                      <CircularProgress size={20} />
-                    </InputAdornment>
-                  ),
-                }}
-              />
-            </FormControl>
-
-            <FormControl>
-              <AddressBookInput name="address" label="Owner" validate={combinedValidate} />
-            </FormControl>
-          </Box>
+          <FormControl fullWidth>
+            <NameInput
+              label="New owner"
+              name="name"
+              placeholder={fallbackName || 'Owner name'}
+              InputLabelProps={{ shrink: true }}
+              InputProps={{
+                endAdornment: resolving && (
+                  <InputAdornment position="end">
+                    <CircularProgress size={20} />
+                  </InputAdornment>
+                ),
+              }}
+            />
+          </FormControl>
+
+          <FormControl fullWidth>
+            <AddressBookInput name="address" label="Owner address or ENS" validate={combinedValidate} required />
+          </FormControl>
+
+          <Divider className={commonCss.nestedDivider} />
+
+          <FormControl fullWidth>
+            <Box display="flex" flexDirection="row" alignItems="center" gap={1} mt={3}>
+              <Typography variant="h6" fontWeight="bold">
+                Threshold
+              </Typography>
+              <SvgIcon component={InfoIcon} color="disabled" inheritViewBox fontSize="small" />
+            </Box>
+
+            <Typography variant="body2" mb={1}>
+              Any transaction requires the confirmation of:
+            </Typography>
+
+            <Grid container direction="row" alignItems="center" gap={1} pt={1}>
+              <Grid item xs={1.5}>
+                <Select value={selectedThreshold} onChange={handleChange} fullWidth>
```

Comment:
Let's use the instance of RHF in this component instead of adding a controlled component.